docs/import_export.rst

```diff
@@ -129,4 +130,27 @@ averageSignal
 Data import
 ===========
 
-FIXME
+Aggregate cell data can be imported into an Ichnaea instance.  For the
+development environment:
+
+1. Download a Differental Cell Export from `Mozilla's Download page
+   <https://location.services.mozilla.com/downloads>`_.  Do not extract it, but
+   keep it in the compressed ``.csv.gz`` format, in the root of the repository.
+
+2. Run the celery workers::
+
+    $ make runcelery
```

Comment:
Because `load_cell_data.py` runs synchronously, we can skip this step. I should have caught this in my commit. Sorry!